packages/dapp/src/components/listinglist/styledComponents.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import styled, { StyledComponentClass } from "styled-components";
+import { colors, fonts, TabComponentProps } from "@joincivil/components";
+
+export const ListingsTabNav = styled.div`
```

Comment:
One more thought...

Should these styles be in `@joincivil/components`?

They are DApp-specific so I can see why they live here, but our components package is pretty much our dapp and our plugin styles. And if future devs or designers update these styles, they would currently have to update them in both the Storybook stories as well as the DApp.

Maybe restoring the `tabs` directory in `@joincivil/components` is the way to go, and we can put the functional components in there, along with these styles, and just export everything from the directory.